inc/azure_uamqp_c/amqp_management.h

```diff
@@ -14,34 +14,34 @@ extern "C" {
 
 #include "azure_c_shared_utility/umock_c_prod.h"
 
-    typedef enum OPERATION_RESULT_TAG
+    typedef enum AMQP_MANAGEMENT_EXECUTE_OPERATION_RESULT_TAG
     {
-        OPERATION_RESULT_OK,
-        OPERATION_RESULT_ERROR,
-        OPERATION_RESULT_OPERATION_FAILED
-    } OPERATION_RESULT;
+        AMQP_MANAGEMENT_EXECUTE_OPERATION_OK,
+        AMQP_MANAGEMENT_EXECUTE_OPERATION_ERROR,
```

Comment:
What's the difference between ERROR and FAILED? Could you add a comment in front to clarify?